### PR TITLE
completed stress testing

### PR DESCRIPTION
**Pull request by morgangrace11**

### CODE REVIEW

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['car-description'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'c725e2c543dbb39aa276a5747cb6e3446b1afb10',
+  logging: {
```
Comment (ccades) [inline newrelic.js]: i may put this on the .gitignore in case that's an important number

File: newrelic.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      'request.headers.setCookie*',
+      'request.headers.x*',
+      'response.headers.cookie',
+      'response.headers.authorization',
+      'response.headers.proxyAuthorization',
+      'response.headers.setCookie*',
+      'response.headers.x*'
+    ]
+  }
+}
```
Comment (ccades) [inline newrelic.js]: if you're using the airBnB linter they like extra lines at the end of every file

File: server/Other Databases/config.cql [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+COPY cars.description (car_id, companyName, carName, edition, trips, mpg, gas, doors, seats, description, business, features, extras, guidelines, faq, moreD, moreE, sentence) FROM 'csv/output6.csv' WITH HEADER = TRUE and delimiter = ',';
+COPY cars.description (car_id, companyName, carName, edition, trips, mpg, gas, doors, seats, description, business, features, extras, guidelines, faq, moreD, moreE, sentence) FROM 'csv/output7.csv' WITH HEADER = TRUE and delimiter = ',';
+COPY cars.description (car_id, companyName, carName, edition, trips, mpg, gas, doors, seats, description, business, features, extras, guidelines, faq, moreD, moreE, sentence) FROM 'csv/output8.csv' WITH HEADER = TRUE and delimiter = ',';
+COPY cars.description (car_id, companyName, carName, edition, trips, mpg, gas, doors, seats, description, business, features, extras, guidelines, faq, moreD, moreE, sentence) FROM 'csv/output9.csv' WITH HEADER = TRUE and delimiter = ',';
+COPY cars.description (car_id, companyName, carName, edition, trips, mpg, gas, doors, seats, description, business, features, extras, guidelines, faq, moreD, moreE, sentence) FROM 'csv/output10.csv' WITH HEADER = TRUE and delimiter = ',';
```
Comment (ccades) [inline server/Other Databases/config.cql]: if you want to combine these theres a way to do it in terminal pretty easily. Lmk if you want some help with it... i made a 10 million seed in terminal myself

File: server/schema.sql [hunk 4]
```diff
@@ -1,34 +1,58 @@
...
+\COPY description FROM 'server/db/output6.csv'
+WITH DELIMITER ',' CSV HEADER;
+\COPY description FROM 'server/db/output7.csv'
+WITH DELIMITER ',' CSV HEADER;
+\COPY description FROM 'server/db/output8.csv'
+WITH DELIMITER ',' CSV HEADER;
+\COPY description FROM 'server/db/output9.csv'
+WITH DELIMITER ',' CSV HEADER;
+\COPY description FROM 'server/db/output10.csv'
+WITH DELIMITER ',' CSV HEADER;
```
Comment (ccades) [inline server/schema.sql]: are all these from the 10 csv files too?

Comment (morgangrace11) [reply server/schema.sql]: for a different database than the others yes

**Pull request merged**